CodeStarterCamp_Week2/main.swift

```diff
@@ -6,7 +6,41 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
-import Foundation
+import Swift
 
-print("Hello, World!")
+let myLottoNumbers: [Int] = [1,2,3,4,5,6]
```

Comment:
```suggestion
let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
```

`,` 사이에는 간격을 주는게 어떨까요?

Reply:
확실히 간격을 주니 훨씬 보기가 수월 하네요.
안좋은 습관을 빨리 고치도록 하겠습니다!

Reply:
https://github.com/yagom-academy/swift-starter-week2/pull/84/commits/68595ee61b0d9aed05425fe1f934b74fb82846bb

간격 및 Array 선언방식 수정했습니다!